### PR TITLE
Fix api for UI

### PR DESCRIPTION
**Pull request by tloubrieu-jpl**
## 🗒️ Summary
Changes made after the integration with the DOI-UI has been tested.

## ⚙️ Test Data and/or Report
You can test reserve and update/submit with following data files:
[reserve2.csv](https://github.com/NASA-PDS/doi-service/files/8248324/reserve2.csv)
[bundle_5.0.xml.zip](https://github.com/NASA-PDS/doi-service/files/8248327/bundle_5.0.xml.zip)

## ♻️ Related Issues

- fixes https://github.com/NASA-PDS/doi-service/issues/326

### CODE REVIEW

File: src/pds_doi_service/core/util/initialize_production_deployment.py [hunk 1]
```diff
@@ -65,16 +65,17 @@
...
 from pds_doi_service.core.outputs.doi_record import CONTENT_TYPE_JSON
 from pds_doi_service.core.outputs.osti.osti_web_parser import DOIOstiXmlWebParser
 from pds_doi_service.core.outputs.service import DOIServiceFactory
 from pds_doi_service.core.outputs.service import SERVICE_TYPE_DATACITE
 from pds_doi_service.core.outputs.service import VALID_SERVICE_TYPES
 from pds_doi_service.core.util.config_parser import DOIConfigUtil
 from pds_doi_service.core.util.general_util import get_logger
-from pds_doi_service.core.util.node_util import NodeUtil
+
+# from pds_doi_service.core.util.node_util import NodeUtil
```
Comment (collinss-jpl) [inline src/pds_doi_service/core/util/initialize_production_deployment.py:78]: Looks like you could just remove this line altogether

**Review by collinss-jpl — APPROVED** [review]
Had one suggestion on logging when --force flag is set, but otherwise changes look good to me!

File: src/pds_doi_service/core/entities/exceptions.py [hunk 2]
```diff
@@ -174,5 +174,6 @@ def raise_or_warn_exceptions(exception_classes, exception_messages, log=False):
 
     if log:
         logger.warning(message_to_raise)
+        logger.info('Since the force flag was used, the previous warning is ignored')
```
[the remark this answers is not in the document]
Comment (tloubrieu-jpl) [reply src/pds_doi_service/core/entities/exceptions.py]: Good catch thanks Scott !

File: src/pds_doi_service/core/util/initialize_production_deployment.py (hunk 1 above)
Comment (tloubrieu-jpl) [reply src/pds_doi_service/core/util/initialize_production_deployment.py:78]: Right, I did it, thanks

**Pull request merged**